YoutubeExplode/Bridge/SearchResponse.cs

```diff
@@ -129,44 +129,48 @@ internal partial class SearchResponse
     public class PlaylistData(JsonElement content)
     {
         [Lazy]
-        public string? Id => content.GetPropertyOrNull("playlistId")?.GetStringOrNull();
+        public string? Id => content.GetPropertyOrNull("contentId")?.GetStringOrNull();
 
         [Lazy]
-        public string? Title =>
-            content.GetPropertyOrNull("title")?.GetPropertyOrNull("simpleText")?.GetStringOrNull()
-            ?? content
-                .GetPropertyOrNull("title")
-                ?.GetPropertyOrNull("runs")
-                ?.EnumerateArrayOrNull()
-                ?.Select(j => j.GetPropertyOrNull("text")?.GetStringOrNull())
-                .WhereNotNull()
-                .ConcatToString();
+        private JsonElement? Metadata => content.GetPropertyOrNull("metadata")?.GetPropertyOrNull("lockupMetadataViewModel");
+
+        [Lazy]
+        public string? Title => Metadata?.GetPropertyOrNull("title")?.GetPropertyOrNull("content")?.GetStringOrNull();
 
         [Lazy]
         private JsonElement? AuthorDetails =>
-            content
-                .GetPropertyOrNull("longBylineText")
-                ?.GetPropertyOrNull("runs")
+            Metadata
+                ?.EnumerateDescendantProperties("metadataParts")
+                ?.ElementAtOrNull(0)
                 ?.EnumerateArrayOrNull()
-                ?.ElementAtOrNull(0);
+                ?.ElementAtOrNull(0)
+                ?.GetPropertyOrNull("text");
 
         [Lazy]
-        public string? Author => AuthorDetails?.GetPropertyOrNull("text")?.GetStringOrNull();
+        public string? Author => AuthorDetails?.GetPropertyOrNull("content")?.GetStringOrNull();
 
         [Lazy]
         public string? ChannelId =>
             AuthorDetails
-                ?.GetPropertyOrNull("navigationEndpoint")
+                ?.GetPropertyOrNull("commandRuns")
+                ?.EnumerateArrayOrNull()
+                ?.ElementAtOrNull(0)
+                ?.GetPropertyOrNull("onTap")
+                ?.GetPropertyOrNull("innertubeCommand")
                 ?.GetPropertyOrNull("browseEndpoint")
                 ?.GetPropertyOrNull("browseId")
                 ?.GetStringOrNull();
```

Comment:
Please keep the original codepaths but add your new alternatives as a fallback, via `??`. Most YouTube changes are rolled out in waves so both codepaths are likely to be in use at the same time.

Reply:
@Tyrrrz Can do - but wouldn't it make more sense (performance-wise) to **pre**pend the new path alternatives instead of appending them? That way you don't check all the (probably broken) old paths before hitting a working one.

Reply:
Yes, it would make more sense that way, thank you